src/scripts/components/time-slider/time-slider.styl

```diff
@@ -2,42 +2,109 @@
 
 $range-height = 5px
 
+thumb()
+  width: 16px
+  height: 16px
+  border-radius: 50%
+  background: $main
+  box-shadow: 0px 1px 5px rgba(0, 0, 0, 0.2), 0px 3px 4px rgba(0, 0, 0, 0.12), 0px 2px 4px rgba(0, 0, 0, 0.14)
```

Comment:
Please make a separate mixin for the box shadow so that we have this declaration only once